src/resources/views/crud/inc/datatables_logic.blade.php

```diff
@@ -50,7 +49,24 @@
 
     @endif
         if (saved_list_url && persistentUrl!=window.location.href) {
+            //before redirecting the user we get the current alerts for this page
+            //we can show them to the user after the persistency redirects him
+            var $passAlerts = [];
+            @foreach (\Alert::getMessages() as $type => $messages)
+
+                @foreach ($messages as $message)
+                    $passAlerts.push({
+                        type: "{{ $type }}",
+                        text: "{!! str_replace('"', "'", $message) !!}"
+                    });
+                @endforeach
+            @endforeach
+            //we setup variables that will be read by our alerts.blade.php file.
+            localStorage.setItem('passAlerts', JSON.stringify($passAlerts));
```

Comment:
`passAlerts` seems a little unnatural as a name. Do you foresee any problems in just calling this `alerts`?